.drone.yml

```diff
@@ -4759,19 +4815,18 @@ steps:
       GOPATH: /go
       OS: linux
       ARCH: amd64
-    settings:
-      username:
-        from_secret: QUAYIO_DOCKER_USERNAME
-      password:
-        from_secret: QUAYIO_DOCKER_PASSWORD
+      AWS_ACCESS_KEY_ID:
+        from_secret: TELEPORT_BUILD_USER_KEY
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: TELEPORT_BUILD_USER_SECRET
     volumes:
       - name: dockersock
         path: /var/run
     commands:
       - apk add --no-cache make
       - chown -R $UID:$GID /go
-      - docker login -u="$PLUGIN_USERNAME" -p="$PLUGIN_PASSWORD" quay.io
-      - docker pull quay.io/gravitational/teleport-buildbox:BUILDBOX_VERSION || true
+      - aws ecr get-login-password --region us-west-2 | docker login -u="AWS" --password-stdin 146628656107.dkr.ecr.us-west-2.amazonaws.com
+      - docker pull quay.io/gravitational/teleport-buildbox:$BUILDBOX_VERSION || true
```

Comment:
:open_mouth: `|| true` biting us

Reply:
It's my understanding that pulling the buildbox is optional, or if it fails to pull we will build it from scratch. In that case I don't think we would want to stop the build based on this failure. It seems like the line could actually just be removed since I think we handle the case of running with drone directly within the Makefile. https://github.com/gravitational/teleport/blob/992a5d5fd40cda7b4129e73b09e858f381f15d5a/build.assets/Makefile#L122-L137

Reply:
Good point -- lets remove this redundant pull then.

Reply:
Removed in https://github.com/gravitational/teleport/pull/14347/commits/c5a32528f3503a6adc4430f18055ebaedb88c93e